src/contracts/core/DelegationManager.sol

```diff
@@ -422,6 +425,71 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
         }
     }
 
+    /**
+     * @notice Called by an avs to register an operator with the avs.
+     * @param operator The address of the operator to register.
+     * @param operatorSignature The signature, salt, and expiry of the operator's signature.
+     */
+    function registerOperatorToAVS(
+        address operator,
+        ISignatureUtils.SignatureWithSaltAndExpiry memory operatorSignature) external onlyWhenNotPaused(PAUSED_OPERATOR_REGISTER_DEREGISTER_TO_AVS) {
```

Comment:
Oh, not blocking but would request that the `external` portion of the method declaration is on a new line. Check out our other multi-line method declarations for an example -- like `_delegate` below

Reply:
ok will do before merging